server/src/main/java/com/coincoinche/engine/CoincheGame.java

```diff
@@ -102,4 +117,16 @@ public Player getPlayer(String username) {
 
     throw new IllegalArgumentException("Player not found with this username");
   }
+
+  /**
+   * Get the current phase of the game, i.e. BIDDING or MAIN.
+   *
+   * @return the current phase.
+   */
+  public Phase getCurrentRoundPhase() {
```

Comment:
Fine for now, but eventually we should get rid of this to decouple game logic from websockets and events (maybe by using a move factory and the move interface because that's the only thing that isn't the same for each phase).